pkg/controller/useraccount/useraccount_controller.go

```diff
@@ -212,38 +221,25 @@ func newUser(userAcc *toolchainv1alpha1.UserAccount) *userv1.User {
 		ObjectMeta: metav1.ObjectMeta{
 			Name: userAcc.Name,
 		},
+		Identities: []string{getIdentityName(userAcc)},
 	}
 	return user
 }
 
-func newIdentity(userAcc *toolchainv1alpha1.UserAccount) *userv1.Identity {
+func newIdentity(userAcc *toolchainv1alpha1.UserAccount, user *userv1.User) *userv1.Identity {
 	name := getIdentityName(userAcc)
 	identity := &userv1.Identity{
 		ObjectMeta: metav1.ObjectMeta{
 			Name: name,
 		},
 		ProviderName:     config.GetIdP(),
 		ProviderUserName: userAcc.Spec.UserID,
-	}
-	return identity
-}
-
-func newMapping(user *userv1.User, identity *userv1.Identity) *userv1.UserIdentityMapping {
-	name := identity.Name
-	mapping := &userv1.UserIdentityMapping{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: name,
-		},
 		User: corev1.ObjectReference{
 			Name: user.Name,
 			UID:  user.UID,
 		},
```

Comment:
@alexeykazakov same here, setting `User` in `newIdentity()` is enough to established mapping from Identity to User.  The logic here [link](https://github.com/codeready-toolchain/member-operator/blob/6be0efb12395e5d45ac450e8fbc0eaa55e012570/pkg/controller/useraccount/useraccount_controller.go#L175-L189) never get executed.

Reply:
Same as above. It's needed for updates.